zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/ZbColumnFamilies.java

```diff
@@ -177,7 +177,8 @@ public enum ZbColumnFamilies implements EnumValue {
 
   MESSAGE_CORRELATION(85),
 
-  USERS(86);
+  USERS(86),
+  USERNAME_BY_USER_KEY(87);
```

Comment:
🔧 The CF name is the wrong way around. The CF stores user keys by username. E.g. we use the username to get data from the CF and the data is the key 😄